src/components/app/app.jsx

```diff
@@ -1,12 +1,41 @@
 import React from "react";
 import PropTypes from "prop-types";
+import {BrowserRouter, Route, Switch} from "react-router-dom";
+
+// pages //
 import WelcomeScreen from "../welcome-screen/welcome-screen";
+import FailScreen from "../fail-screen/fail-screen";
+import GameArtistScreen from "../game-artist-screen/game-artist-screen";
+import GameGenreScreen from "../game-genre-screen/game-genre-screen";
+import LoginScreen from "../login-screen/login-screen";
+import SuccessScreen from "../success-screen/success-screen";
 
 const App = (props) => {
   const {errorsCount} = props;
 
   return (
-    <WelcomeScreen errorsCount={errorsCount} />
+    <BrowserRouter>
+      <Switch>
+        <Route path="/" exact>
+          <WelcomeScreen errorsCount={errorsCount} />
+        </Route>
+        <Route path="/signin" exact>
```

Comment:
В задании адрес маршрута должен быть другим, это важно 

```suggestion
        <Route path="/login" exact>
```